bedrock/mozorg/redirects.py

```diff
@@ -578,10 +581,10 @@ def decider(request, **kwargs):
     # bug 1233015
     redirect(r'^about/partnerships/contentservices(/.*)?$', 'mozorg.partnerships'),
 
+    # Bug 1235853
+    redirect(r'^facebookapps(/.*)?$', 'firefox.new'),
+
     # bug 832348 **/index.html -> **/
     # leave this at the bottom
```

Comment:
Note: I'm also moving the `facebookapps` redirect above this line, as I only just noticed this comment.